docs/integrations/node.md

```diff
@@ -1,3 +1,53 @@
 # Node
 
-TODO
+The `nodeMedicusPlugin` is a plugin that adds a health check for the Node process, with optional system diagnostics. It provides detailed system information, such as CPU usage, memory stats, load averages, and uptime when the `debug` flag is enabled (which is set to `true` by default).
+
+To integrate the `nodeMedicusPlugin` into your Medicus setup, simply import and register it with your Medicus instance:
+
+```ts
+import { Medicus } from 'medicus';
+import { nodeMedicusPlugin } from 'medicus/node';
+
+const medicus = new Medicus({
+  plugins: [nodeMedicusPlugin()]
+});
+```
+
+## Options 
+
+The `nodeMedicusPlugin` accepts the following options: 
+
+ -  **`checkerName`**: The name of the checker. Defaults to `node`.
+ -  **`debug`**: If the output should show detailed system information. Defaults to `true`.
```

Comment:
```suggestion
 -  **`debug`**: Controls if this plugin should show detailed system information on a debug health request. Defaults to `true`. Users might wanna disable this in case showing this kind of detailed information is not needed on debug requests.
```